snap/snapcraft.yaml

```diff
@@ -99,6 +84,29 @@ parts:
       install -DT "./LICENSE" \
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/device-mqtt/LICENSE"
 
-  config-common:
+  runtime-helpers:
     plugin: dump
     source: snap/local/runtime-helpers
+
+  metadata:
+    plugin: nil
+    source-type: git
+    override-pull: |
+      # download the metadata in AppStream format
+      # this is temp URL pending PR https://github.com/canonical/edgex-snap-metadata/pull/4
+      wget https://raw.githubusercontent.com/farshidtz/edgex-snap-metadata/appstream/edgex-device-mqtt.metainfo.xml
```

Comment:
This will be updated once this gets merged:
-  https://github.com/canonical/edgex-snap-metadata/pull/4